lib/generators/test_unit/observer/observer_generator.rb

```diff
@@ -0,0 +1,14 @@
+require "rails/generators/test_unit"
+
+module TestUnit
+  module Generators
+    class ObserverGenerator < ::TestUnit::Generators::Base
+      check_class_collision suffix: "ObserverTest"
+      source_root ::File.expand_path("../templates", __FILE__)
+
+      def create_test_file
+        template "unit_test.rb", ::File.join("test/unit", class_path, "#{file_name}_observer_test.rb")
```

Comment:
__[rubocop]__
* Line is too long. [102/100] [:ref](https://github.com/bbatsov/ruby-style-guide#80-character-limits)

<!-- Comment by [SideCI](https://sideci.com) -->